testsuite/integration/elytron/pom.xml

```diff
@@ -281,11 +285,44 @@
                         <jboss.server.config.file.name>standalone-full.xml</jboss.server.config.file.name>
                         <jboss.inst>${wildfly.dir}</jboss.inst>
                         <module.path>${wildfly.dir}/modules</module.path>
-                        <server.jvm.args>-Dmaven.repo.local=${settings.localRepository} -Djboss.bind.address=${node0} -Djboss.bind.address.management=${node0} ${surefire.system.args} ${jvm.args.ip.server} ${jvm.args.other} ${jvm.args.timeouts} ${extra.server.jvm.args}</server.jvm.args>
+                        <server.jvm.args>${server.jvm.args}</server.jvm.args>
                         <node0>${node0}</node0>
+                        <security.properties>${security.properties}</security.properties>
                     </systemPropertyVariables>
-                    <argLine>${surefire.system.args} ${jvm.args.ip.client}</argLine>
                 </configuration>
+                <executions>
+                    <execution>
+                        <id>default-tests</id>
+                        <phase>test</phase>
+                        <goals>
+                            <goal>test</goal>
+                        </goals>
+                        <configuration>
+                            <skip>false</skip>
+                            <argLine>${surefire.system.args} ${jvm.args.ip.client}</argLine>
+                            <excludes>
+                                <exclude>***/UndertowSSLv2HelloTestCase</exclude>
+                            </excludes>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>sslv2-tests</id>
+                        <phase>test</phase>
+                        <goals>
+                            <goal>test</goal>
+                        </goals>
+                        <configuration>
+                            <skip>false</skip>
+                            <argLine>-Djava.security.properties==${security.properties} ${surefire.system.args} ${jvm.args.ip.client}</argLine>
```

Comment:
If we could use the single `=` here to append to the existing properties file instead of fully override, we wouldn't need to specify the security providers in the properties file.

Reply:
Fixed :)